opnode/test/system_test.go

```diff
@@ -479,6 +481,30 @@ func L1InfoFromState(ctx context.Context, contract *l1block.L1Block, l2Number *b
 	return out, nil
 }
 
+// TestSystemMockP2P sets up a L1 Geth node, a rollup node, and a L2 geth node and then confirms that
+// the nodes can sync L2 blocks before they are confirmed on L1.
+func TestSystemMockP2P(t *testing.T) {
+	if !verboseGethNodes {
+		log.Root().SetHandler(log.DiscardHandler())
+	}
+
+	cfg := defaultSystemConfig(t)
+	// slow down L1 blocks so we can see the L2 blocks arrive before the L1 blocks do.
+	cfg.L1BlockTime = 10
+	// connect the nodes
+	cfg.P2PTopology = map[string][]string{
+		"verifier": []string{"sequencer"},
+	}
+
+	sys, err := cfg.start()
+	require.Nil(t, err, "Error starting up system")
+	defer sys.Close()
+
+	t.Log("successfully set up network")
```

Comment:
For later. We should add a p2p management API. It's also be good to use it to check that nodes can peer.